libbeat/docs/security/users.asciidoc

```diff
@@ -0,0 +1,295 @@
+[role="xpack"]
+[[feature-roles]]
+=== Grant users access to secured resources
+
+//TODO: Redirect beats-user-access to feature-roles to prevent 404s.
+
+// REVIEWERS: Our built-in roles don't provide enough granularity for some users,
+// and in some cases, the roles don't cover all the tasks that a single user
+// might perform (such as running the setup). Plus the roles don't grant access
+// to Beats indices. I list privileges here (rather than roles) so that the person
+// setting up security has more flexibility and the knowledge they need to
+// create their own roles, if necessary. Is this a likely scenario?
```

Comment:
Yes, I think this makes sense. Builtin roles are not really meant for setup or ingesting data only with Beats. Plus actual scenario/privileges required might depend on environment and how Beats is configured/rolled out.